### PR TITLE
Little christmas

### PR DESCRIPTION
**Pull request by thevangelist**
@jkpl do you think this information should be public?

...Current implementation is missing a page at `/meetings`. should we add one?

### CODE REVIEW

File: resources/pages/meetings/2016/index.html [hunk 1]
```diff
@@ -0,0 +1,39 @@
+{:description "Little Christmas v0.0.1"}
```
Comment (jkpl) [inline resources/pages/meetings/2016/index.html]: Needs a `:title` as well.

**Comment by jkpl** [comment]
re `/meetings`: yeah we can add one if you like

**Comment by thevangelist** [comment]
@jkpl can you add a basic listing of events (where topmost = latest)

File: resources/pages/meetings/2016/index.html (hunk 1 above)
Comment (thevangelist) [inline resources/pages/meetings/2016/index.html]: Added

**Comment by jkpl** [comment]
@thevangelist Sure, I'll whip up something for that. I'll probably try to make it a bit generic, so that we can easily include a similar listing page for other categories of pages later.

**Pull request merged**